rules/no-unknown-modifiers.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+var createAvaRule = require('../create-ava-rule');
+
+var modifiers = {
+	after: true,
+	afterEach: true,
+	before: true,
+	beforeEach: true,
+	cb: true,
+	only: true,
+	serial: true,
+	skip: true,
+	todo: true
+};
+
+function getTestModifiers(node) {
+	if (node.type === 'CallExpression') {
+		return getTestModifiers(node.callee);
+	}
```

Comment:
Sure thing, done ;)
I'd say make it a xo rule, but it's not a rule that I like much, and I'm starting to use xo more and more, so I won't say anything ;)